src/api.ts

```diff
@@ -92,9 +92,11 @@ export class OpsLevelGraphqlAPI implements OpsLevelApi {
     return this.client.request(query, { alias: serviceAlias }, { "GraphQL-Visibility": "internal" });
   }
 
-  getServicesReport() {
+  getServicesReport(includeScorecards: boolean) {
     const query = gql`
-      query servicesReport {
+      query servicesReport(
+        $includeScorecards: Boolean
```

Comment:
🧠 Is there a scenario where we *don't* want this?

I can't think of one, but maybe I'm just not forward looking enough.